src/pymor/operators/constructions.py

```diff
@@ -1362,3 +1362,55 @@ def __call__(self, U, mu=None):
         if self.raise_negative and np.any(norm_squared < 0):
             raise ValueError(f'norm is negative (square = {norm_squared})')
         return np.sqrt(norm_squared)
+
+
+class NumpyConversionOperator(Operator):
+    """Converts |VectorArrays| to |NumpyVectorArrays|.
+
+    Note that the input |VectorArrays| need to support
+    :meth:`~pymor.vectorarrays.interface.from_numpy`.
+
+    Parameters
+    ----------
+    space
+        The |VectorSpace| of the |VectorArrays| that are converted to
+        |NumpyVectorArrays|.
+    direction
+        Either `'to_numpy'` or `'from_numpy'`. In case of `'to_numpy'`
+        :meth:`apply` takes a |VectorArray| from `space` and returns
+        a |NumpyVectorArray|. In case of `'from_numpy'`, :meth:`apply`
+        takes a |NumpyVectorArray| and returns a |VectorArray| from
+        `space`.
+    """
+
+    linear = True
+
+    def __init__(self, space, direction='to_numpy'):
+        assert direction in ('to_numpy', 'from_numpy')
+        self.__auto_init(locals())
+        if direction == 'to_numpy':
+            self.source = space
+            self.range = NumpyVectorSpace(space.dim, id=space.id)
+        else:
+            self.range = space
+            self.source = NumpyVectorSpace(space.dim, id=space.id)
```

Comment:
```suggestion
            self.source = NumpyVectorSpace(space.dim, id=space.id)
            self.range = space
```
I think this is a better way to sort this for readability.